modules/frontend/search_sharder.go

```diff
@@ -23,6 +24,8 @@ import (
 const (
 	defaultTargetBytesPerRequest = 100 * 1024 * 1024
 	defaultConcurrentRequests    = 1000
+	// every search is broken into the same number of search shards. these shards are used to stream partial results back that are guaranteed to be latest results
+	maxSearchShards = 200
```

Comment:
Worth making this a configurable tunable?

Reply:
I was trying to avoid tunables, but yes :). will do